src/output.cpp

```diff
@@ -675,24 +685,21 @@ std::string TextOutput::map_key_to_str(BPFtrace &bpftrace,
   return map.name() + map_key.argument_value_list_str(bpftrace, key);
 }
 
-std::string TextOutput::map_keyval_to_str(const SizedType &map_type,
-                                          const std::string &key,
-                                          const std::string &val) const
+void TextOutput::map_keyval(const SizedType &map_type,
```

Comment:
naming nit to match the other snake case funcs 
```suggestion
void TextOutput::map_key_val(const SizedType &map_type,
```